pyscript.core/docs/README.md

```diff
@@ -281,6 +281,24 @@ This really reflects how otherwise native Web inline events handlers work and we
 
 ℹ️ - Please note that if your code runs via *XWorker*, hence in a different thread, there are different caveats and constraints to consider. Please read the [XWorker](#xworker) dedicated section to know more.
 
+
+#### The direct method / function
+
+If the content of the generic *event* attribute is just a function name (that means: no parenthesis, no `event` passed along, just `print` or `my_method` or `my_function` name), *core* tries to grab that from the current execution context and invokes it directly passing the *event*:
+
+```html
+<script type="micropython">
+    def print_type(event):
+        print(event.type)
+</script>
+<button micropython-click="print_type">
+    print type directly
+</button>
+```
+
+The advantage of this alternative is that *core* does not need to temporarily create an event at all, the disadvantage is that it's not possible to pass any other value if not just the currently triggered *event*.
```

Comment:
What do you mean by this:

> the disadvantage is that it's not possible to pass any other value if not just the currently triggered *event*.

Reply:
`py-click="foo(event, bar + 1)"` won't be possible as there's no evaluation wanted/needed/desired with the `py-click="foo"` approach and I don't think there should ever be even a proposal to allow "*space or comma separated values*" such as `py-click="foo bar+1"` as that opens a universe of issues and would require evaluation defeating completely the goal/scope of this resolution around the topic.

Reply:
I see, thanks for explaining! In this example `bar` has to have been a global variable anyways? So it's at least accessible, though you would have to have an adaptor:
```py
def handler(event):
   # bar here is in global scope
   foo(event, bar + 1)
```
So it's a bit less convenient, hence the disadvantage. 

But as I said I don't want to get into arguing the pros/cons of different APIs here. I think there are a lot of different reasonable solutions and if other people want to take a side they are welcome to do so =)

Reply:
yes, your adapter would work without any issue ... this PR goal is to address not just directly exposed globals, but solve the future case when an interpreter can't evaluate code (those just targeting WASM that only expose a method and nothing else). The current logic addresses all cases and it's documented and tested to prove it works, so I think we should *just* move forward and offer this extra feature (imho).

Reply:
TL;DR if anyone approves this I'll be able to write integration tests around this and, like I've said, there's never ambiguity moving this forward + we can think about issues whenever these exist, not out of imagination ;-)

Reply:
P.S. bar doesn't need to be global ... it was just an example around the fact I don't think we should re-invent a new syntax here, and move forward with this proposal so that all edge cases can be addressed in code, as opposite of needing to be addressed in *our* project. Let devs do dev things and let us enable edge case ... more than that it's a slippery slope to me.